projects/odf_transform/odf_transform/config.json

```diff
@@ -28,5 +28,11 @@
         "keywords_vocabulary": "GCMD Science Keywords",
         "Conventions": "CF-1.6,CF-1.7,ACDD1.1",
         "comment": ""
+    },
+    "flag_convention": {
```

Comment:
This doesn't seem like user configuration to me, should probably go into the code and keep this file to things that will change with each dataset

Reply:
It may change from one organization to another, maybe also between datatypes. I don't really know yet if all ODF used the same convention or not. Making it clear for now in the config file is at least the best way for now I think to make sure we're using the right one. That's my thought. If it is standard for ODFs we can definitely stick it in odf_parser

Reply:
I made a typo correction to the flag_meanings attribute